src/package.rs

```diff
@@ -84,3 +84,13 @@ pub fn prepare_package(package_url: String, package_archive: String, output_dire
     }
     Ok(())
 }
+
+pub fn remove_package(package_archive: String, output_directory: String) -> Result<()> {

```

Comment:
I would suggest using slices here, so you won't need to use `.to_string()` for literals everytime calling the funciton
```suggestion
pub fn remove_package(package_archive:  &str, output_directory:  &str) -> Result<()> {
```

All `fs::` functions accept `&str`

Reply:
Thank you. Good idea. Correction applied.